erpnext/stock/doctype/delivery_note/test_delivery_note.py

```diff
@@ -321,12 +319,58 @@ def test_return_for_serialized_items(self):
 		dn.cancel()
 		
 		self.check_serial_no_values(serial_no, {
-			"status": "Available",
 			"warehouse": "_Test Warehouse - _TC",
 			"delivery_document_no": "",
 			"purchase_document_no": se.name
 		})
+		
+	def test_delivery_of_bundled_items_to_target_warehouse(self):
```

Comment:
In this test, we have passed the target warehouse as `_Test Warehouse 1 - _TC`, but we haven't changed the target warehouse in bundled item to something different. Isn't that the requirement of #3970 ?